go.mod

```diff
@@ -14,6 +14,7 @@ require (
 	github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/privatedns/armprivatedns v0.2.0
 	github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/resources/armresources v0.2.0
 	github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/storage/armstorage v0.2.0
+	github.com/Azure/azure-sdk-for-go/sdk/storage/azblob v0.2.0
```

Comment:
Chan you revert thoses diff on the mod, there is no reason to change that here

Reply:
It's the result of a go mod tidy, it's probably that the go module file is still formatted with go1.16, not go1.17. I created a single commit for it but can revert it.